### PR TITLE
Add landing page

### PR DESCRIPTION
**Pull request by jreinlein**
Fixes issue #159 

I took a stab at adding useful info to the landing page (default `/` path). Let me know what you think! **Very open to suggestions.**

### Testing
- Clicked around, looked at console for errors
- `npm run test` and `npm run lint`

### [🎦  Screen recording](https://drive.google.com/file/d/121QF4KAVzDBGpw8Oo7o_dbvJggBA6KSM/view?usp=sharing)
(GitHub doesn't accept `.mov` or `.mp4` 😠  ...any suggestions for how to upload recordings?)

### CODE REVIEW

File: src/webapp/pages/Home.js [hunk 1]
```diff
@@ -1,7 +1,98 @@
...
+  icon: {
+    marginRight: "0.5rem",
+  },
+  subAppPaperLink: {
+    fontSize: "1.5rem",
+    display: "flex",
+    alignItems: "center",
+    justifyContent: "center",
+    "&:hover": {
+      backgroundColor: "hsla(0, 0%, 0%, 0.05)",
```
Comment (piratefsh) [inline src/webapp/pages/Home.js:25]: minor nit: i think the bg color can be a _tad_ darker for better contrast on brighter screens.

**Review by piratefsh — APPROVED** [review]
it looks so good!! i think the way you have it organized makes sense, plus having the links really adds context. one minor comment but otherwise lgtm!

re: screengrabs, i think linking out is fine. i usually make gifs with Giphy and upload them to the PR but it does limit the length of the demo.

File: src/webapp/pages/Home.js (hunk 1 above)
Comment (jreinlein) [reply src/webapp/pages/Home.js:25]: @piratefsh you want a darker colour when hovering over the Papers that link to the other apps? Or do you mean the actual bg colour outside of the Papers?

Comment (piratefsh) [reply src/webapp/pages/Home.js:25]: ah I meant the actual BG outside of the Paper! maybe increase the opacity on backgroundColor here. feel free to tweak to see what looks good.

Comment (jreinlein) [reply src/webapp/pages/Home.js:25]: Modifying the background color would require [affecting the root](https://stackoverflow.com/questions/48186478/custom-theme-background-color) styling which would affect each page as far as I can tell, so as a compromise I've increased the `elevation` used in the Papers

image

**Pull request merged**